src/disco/components/Addon.js

```diff
@@ -225,6 +228,18 @@ export class AddonBase extends React.Component {
       extension: type === ADDON_TYPE_EXTENSION,
     });
 
+    if (!addon) {
+      return (
+        <div className={addonClasses}>
+          <div className="content">
+            <div className="copy">
+              <LoadingText />
```

Comment:
Yes, exactly. No one will see it. The style isn't great but I fixed it up a little. I just needed the component to return early for not-yet-loaded add-ons to avoid throwing an exception.